webui/src/pages/repositories/repository/actions/index.jsx

```diff
@@ -174,8 +161,11 @@ const ActionsList = ({ repo, after, onPaginate, branch, commit, onFilterBranch,
                     <RefreshButton onClick={doRefresh}/>
                 </ActionGroup>
             </ActionsBar>
-
             {content}
+            <div>
+                {/* eslint-disable-next-line react/jsx-no-target-blank */}
+                Actions can be configured to run when predefined events occur. <a href="https://docs.lakefs.io/setup/hooks.html" target="_blank">Learn more.</a>
```

Comment:
This is intended _always_ to show up, even when there are actions?

Reply:
Yes